src/main/java/net/fabricmc/loom/task/MigrateMappingsTask.java

```diff
@@ -179,7 +203,7 @@ private static void migrateMappings(Project project, MinecraftMappedProvider min
 		 * Since we only use intermediary names (and not descriptors) to match, and intermediary names are unique,
 		 * this will migrate methods that have had their signature changed too.
 		 */
-		public MappingsJoiner(TinyTree sourceMappings, TinyTree targetMappings, String fromNamespace, String toNamespace) {
+		MappingsJoiner(TinyTree sourceMappings, TinyTree targetMappings, String fromNamespace, String toNamespace) {
```

Comment:
Is this used outside of this class?

Reply:
Nope

Reply:
It can be private then

Reply:
The class, I think? the constructor maybe not

Reply:
Nope, might as well make the access weaker for the entire class.